Makefile.pre.in

```diff
@@ -1776,6 +1783,7 @@ PYTHON_HEADERS= \
 		$(srcdir)/Include/cpython/pthread_stubs.h \
 		$(srcdir)/Include/cpython/pyatomic.h \
 		$(srcdir)/Include/cpython/pyatomic_gcc.h \
+		$(srcdir)/Include/cpython/pyatomic_msc.h \
```

Comment:
Isn't this Windows only? If so, we can probably discard it here in the \*nix build.

Reply:
I agree I was probably overzealous. I'm happy to have some extra dependencies, false positives being better than false negatives in this case. I'll correct my overzealousness.

Reply:
Rhetorical question: Why don't we use GCC, CLANG, or other compilers' ability to generate include dependencies? I'm thinking stuff like `gcc -M` or `clang -M`.

Reply:
> Rhetorical question: Why don't we use GCC, CLANG, or other compilers' ability to generate include dependencies? I'm thinking stuff like `gcc -M` or `clang -M`.

I _guess_ just because the current build system has grown into something that is very hard to maintain, so people in general just try to make as few changes as possible, if any at all. Not to mention `configure.ac`.